src/main/fabric8/deployment.yml

```diff
@@ -0,0 +1,21 @@
+metadata:
+  labels:
+    app: ${project.artifactId}
+    project: ${project.artifactId}
+    version: ${project.version}
+spec:
+  template:
+    spec:
+      volumes:
+        - name: config
+          configMap:
+            name: ${project.artifactId}
+            items:
+              - key: application.properties
+                path: application.properties
+      containers:
+        - volumeMounts:
+            - name: config
+              mountPath: /deployments/config
+      imagePullSecrets:
+        - name: ${docker.registry.push.secret-name}
```

Comment:
Is the secret mentioned somewhere? Should be in the docs
And missing newline

Reply:
Fixed in upcomming commit